<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileListPanel</class>
 <widget class="QWidget" name="FileListPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="listView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addFileBtn">
       <property name="text">
        <string>Add files...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveAllBtn">
       <property name="text">
        <string>Save all</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
